From 4dab04622a6cf2a31da26a51dedb60ec44ac89a4 Mon Sep 17 00:00:00 2001
From: Ben Clayton <bclayton-at-google.com@ffmpeg.org>
Date: Fri, 10 Jul 2020 16:53:12 +0100
Subject: [PATCH] libavfilter/glslang: Remove unused header

The <glslang/Include/revision.h> include was not used, and revision.h has
been removed from glslang master.
See: https://github.com/KhronosGroup/glslang/pull/2277
---
 libavfilter/glslang.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/libavfilter/glslang.cpp b/libavfilter/glslang.cpp
index 497df6e..125eb5c 100644
--- a/libavfilter/glslang.cpp
+++ b/libavfilter/glslang.cpp
@@ -24,7 +24,6 @@ extern "C" {
 }
 
 #include <glslang/Include/ResourceLimits.h>
-#include <glslang/Include/revision.h>
 #include <glslang/Public/ShaderLang.h>
 #include <glslang/SPIRV/GlslangToSpv.h>
 
-- 
2.7.4

